app/views/content/is-teaching-right-for-me/computing/_article.html.erb

```diff
@@ -44,7 +46,9 @@ image_2_alt: "",
 <li class="tag">Data representation</li>
 <li class="tag">Digital technology</li>
 </ul>
-<p>For 16 to 18 year olds (key stage 5), you'll teach computer science as an A level qualification, helping pupils to develop websites and prepare them for further learning or employment. Themes you might also cover include:</p>
+<p>When you're teaching key stage 4 you'll teach pupils ways to protect their online privacy and identity.</p>
+<p><a href="https://www.thenational.academy/teachers/programmes/computing-secondary-ks4-l/units">Explore what teaching a key stage 4 computing lesson would be like</a>.</p>
+<p>Themes you'll cover when you teach 16 to 18 year olds (key stage 5), A level qualification, include:</p>
```

Comment:
```suggestion
<p>Themes you'll cover when you teach 16 to 18 year olds (key stage 5) computer science as an A level qualification include:</p>
```
do we need to expand this to include `computer science` as per GCSE line?

Reply:
@gemmadallmandfe I wasn't sure if it was included in GCSE because it's broader and called computer science whereas for A level it's computing? But I'll see how it's referred to in the national curriculum/ lesson plans.